src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -429,6 +433,11 @@ case class CHeader(
                   ) extends Header {
 }
 
+object CHeader {
+  val VotesSize = 3
```

Comment:
It's think it's more like `BoxId.size` kind of constant (see https://github.com/ScorexFoundation/sigmastate-interpreter/blob/8e21c7d5f591d640f78210b0190d8948aaa568e0/src/main/scala/org/ergoplatform/ErgoBox.scala#L95-L95 ) 
Do you think it would be a good fit in `ErgoConstants`?
Added comments.

Reply:
Yes, it is better to put ALL constants in a single table. For some of them we need to put NOTE in comments that that should never be changed. 
We are going to use that table later to generate table in the specification.

Reply:
Makes sense. Done.